docs_user/assemblies/assembly_adopting-the-data-plane.adoc

```diff
@@ -0,0 +1,23 @@
+[id="adopting-data-plane_{context}"]
+
+:context: adopting-data-plane
+
+= Adopting the data plane
+
+Adopting the {rhos_long} data plane involves the following steps:
+
+* Stopping any remaining services on the {rhos_prev_long} ({OpenStackShort}) control plane.
```

Comment:
Should this be an ordered list? It sounds like you have to stop the remaining services first.